src/adapters/store.js

```diff
@@ -5,35 +5,59 @@ import { getKey } from 'src/libs/pois';
 import { fire } from 'src/libs/customEvents';
 import { isPoiCompliantKey } from 'src/libs/pois';
 
-async function get(k) {
+const prefix = `qmaps_v${version}_`;
+
+function get(k) {
   try {
-    return JSON.parse(localStorage.getItem(k));
+    const prefixedKey = `${prefix}${k}`;
+    return JSON.parse(localStorage.getItem(prefixedKey));
   } catch (e) {
     Error.sendOnce('local_store', 'get', `error parsing item with key ${k}`, e);
     return null;
   }
 }
 
-async function set(k, v) {
+function set(k, v) {
   try {
-    localStorage.setItem(k, JSON.stringify(v));
+    const prefixedKey = `${prefix}${k}`;
+    localStorage.setItem(prefixedKey, JSON.stringify(v));
   } catch (e) {
     Error.sendOnce('local_store', 'set', 'error setting item', e);
   }
 }
 
+function del(k) {
+  const prefixedKey = `${prefix}${k}`;
+  localStorage.removeItem(prefixedKey);
+}
+
+/**
+ * List keys without prefix.
+ * In case some keys are not prefixed, we don't return them
+ */
+function listKeys() {
+  return Object
+    .keys(localStorage || {})
+    .reduce((acc, k) => {
+      const parts = k.split(prefix);
+      return parts.length === 2
+        ? [...acc, parts[1]]
+        : acc;
+    }, []);
+}
+
 export async function getAllFavorites() {
-  let localStorageKeys = [];
+  let keys = [];
   try {
-    localStorageKeys = Object.keys(localStorage);
+    keys = listKeys();
```

Comment:
I think we can safely remove the try/catch block here. Wdyt ?

Reply:
Has long as you try to access local storage, it's a good idea to use try/catch. Local storage has some weird error cases, for example in some browsers in certain security settings it can exist on `window` while still creating errors.
Some ref https://michalzalecki.com/why-using-localStorage-directly-is-a-bad-idea/ 
I'm not sure the rest of the code isn't completely safe against that, but this try/catch is useful eand doesn't cost us anythin.